UKHO.SalesCatalogueFileShareServicesMock.API/UKHO.SalesCatalogueFileShareServicesMock.API/Controllers/SalesCatalogueServiceController.cs

```diff
@@ -78,6 +78,11 @@ public IActionResult ProductVersions(List<ProductVersionRequest> productVersionR
                 bool isInitalIndex = true;
                 foreach (var item in productVersionRequest)
                 {
+                    //code added to handle 304 not modified scenario
+                    if (item.ProductName == "DE416040" && item.EditionNumber == 11 && item.UpdateNumber == 1)
```

Comment:
Please put these magic values into `const`s so that they are more visible:

```
const string NotMofifiedProductName = "DE416040";
const int NotModifiedEditionNumber = 11;
const int NotModifiedUpdateNumber = 1;
...
if (item.ProductName == NotMofifiedProductName && item.EditionNumber == NotModifiedEditionNumber && item.UpdateNumber == NotModifiedUpdateNumber)
```

Reply:
Thanks for pointing. Made changes accordingly.